tools/dev-doctor/rootcmd.go

```diff
@@ -187,7 +187,8 @@ func rootCmdRun(cmd *cobra.Command, args []string) {
 				ifNotFound:    checkError,
 				versionArgs:   []string{"--version"},
 				versionRegexp: regexp.MustCompile(`hub\s+version\s+(\d+.\d+\.\d+)`),
-				minVersion:    &semver.Version{Major: 2, Minor: 0, Patch: 0},
+				minVersion:    &semver.Version{Major: 2, Minor: 14, Patch: 0},
+				hint:          `Download the latest version from https://github.com/github/hub/releases.`,
```

Comment:
This doesn't seem to have installation steps. Perhaps this link - https://github.com/github/hub? I'll leave it to you.

Reply:
The https://github.com/github/hub link actually contains _incorrect_ installation steps for us. It recommends installing `hub` with your distro's package manager, which will generally get you a version of `hub` that's too old.